tests/cli/test_buildspec.py

```diff
@@ -123,6 +123,9 @@ def test_func_buildspec_find():
     # buildtest buildspec find --quiet
     cache.print_buildspecs(quiet=True)
 
+    # buildtest buildspec find --row-count
+    cache.print_buildspecs(row_count=True)
```

Comment:
can you add test coverage when querying row-count with other options `-b`, `-t`, `-e`

Reply:
take a look at https://app.codecov.io/github/buildtesters/buildtest/pull/1377 

![Screen Shot 2023-02-09 at 12 26 56 PM](https://user-images.githubusercontent.com/12942230/217890980-2e2d4fda-fa36-4d62-ad0a-20d7c2046c4a.png)

Reply:
@shahzebsiddiqui Okay done. I have added the test coverage.